data/_ui/tooltips.txt

```diff
@@ -964,67 +964,67 @@ tip "firing fuel / shot:"
+	`Shield damage dealt to your ship over time by this weapon per shot.`
 
 tip "firing corrosion / shot:"
-	`Each shot applies corrosion to the ship, reducing the ship's hull over time. Corrosion wears off over time.`
+	`Hull damage dealt to your ship over time by this weapon per shot.`
 
 tip "firing leak / shot:"
-	`Each shot causes the ship's fuel to leak. Fuel leakage wears off over time.`
+	`Fuel damage dealt to your ship over time per shot.`
 
 tip "firing burn / shot:"
-	`Each shot causes the ship to burn, increases the ship's heat over time. Burn wears off over time.`
+	`Heat damage dealt to your ship over time per shot.`
 
 tip "% firing energy / shot:"
-	`Each shot reduces the ship's energy as a percentage of the ship's total energy capacity by this amount.`
+	`Energy consumed per shot, as a percentage of the ship's total energy capacity.`
 
 tip "% firing heat / shot:"
-	`Each shot fired increases the ship's heat as a percentage of the ship's maximum heat capacity by this amount.`
+	`Heat generated per shot, as a percentage of the ship's maximum heat capacity.`
 
 tip "% firing fuel / shot:"
-	`The amount of fuel used or lost per shot fired as a percentage of the ship's total fuel.`
+	`Fuel used or lost per shot, as a percentage of the ship's total fuel.`
 
 tip "% firing hull / shot:"
-	`Each shot fired by this weapon causes this amount of hull damage as a percentage of the ship's total hull.`
+	`Hull damage dealt to your ship per shot, as a percentage of your total hull strength.`
 
 tip "% firing shields / shot:"
-	`Each shot fired by this weapon causes this amount of shield loss as a percentage of the ship's total shields.`
+	`Shield damage dealt to your ship per shot, as a percentage of your total shield strength.`
 
 tip "inaccuracy:"
-	`Maximum inaccuracy of this weapon, in degrees.`
+	`Maximum inaccuracy of this weapon, in degrees. A value of one means the weapon's aim will be at most one degree off-target.`
 
 tip "blast radius:"
-	`When this weapon hits something, everything within this distance (including friendly ships) will be damaged.`
+	`Everything within this distance (including friendly ships) will be damaged when the weapon hits something.`
 
 tip "missile strength:"
-	`How resistant this missile is to anti-missile shots.`
+	`This missile's resistance to anti-missile shots.`
 
 tip "anti-missile:"
-	`How powerful this anti-missile system's shots are. Some missiles are harder to destroy than others.`
+	`The power of the anti-missile shots. Higher values make it easier to destroy stronger missiles.`
 
 
 
 # Flight checks:
 tip "overheating!"
-	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You must either add more cooling or remove some of the outfits that are producing heat.`
+	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You should either add more cooling or remove some of the outfits that are producing heat.`
```

Comment:
Anything ending in an exclamation point is something you must resolve in order to actually leave the outfitter, so I think saying "must" instead of "should" for such tooltips makes sense.

Reply:
```suggestion
	`This ship's outfits are producing so much heat that it will overheat the moment it takes off. You must either add more cooling or remove some of the outfits that are producing heat.`
```